src/main/java/org/fogbowcloud/manager/core/models/token/Token.java

```diff
@@ -1,5 +1,189 @@
 package org.fogbowcloud.manager.core.models.token;
 
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+
+import org.fogbowcloud.manager.core.utils.DateUtils;
+import org.fogbowcloud.manager.core.utils.JSONHelper;
+import org.json.JSONException;
+import org.json.JSONObject;
+
 public class Token {
+	
+	private static final String DATE_EXPIRATION = "dateExpiration";
```

Comment:
Rename to `EXPIRATION_DATE = "expirationDate"`.

Reply:
Done :). Check [this](https://github.com/fogbow/fogbow-manager-core/commit/8aba75e25659c73671a6bbc0feac76d351d6a70f) out.